app/ui/src/app/dashboard/dashboard.component.ts

```diff
@@ -1,7 +1,8 @@
 import { Component, OnInit } from '@angular/core';
 import { Observable } from 'rxjs/Observable';
 
-import { Connections, Integrations } from '../model';
+import { Connections } from '../model';
+import { Integrations } from '@syndesis/ui/integration';
 import { ConnectionStore } from '../store/connection/connection.store';
 import { IntegrationStore } from '../store/integration/integration.store';
```

Comment:
I do believe that you can combine these two into a single import statement, like:

```
import { ConnectionStore, IntegrationStore } from '@syndesis/ui/store';
``` 

I just doubled checked those folders and the barrels were already in place.

Reply:
Yeah, both of these are two cases where I kinda debated if I should go and touch the other imports or kinda try and keep the changes a bit more focused.  I think since we'll eventually also be moving or replacing these stores I think we could leave this for now and fix this line when the stores get dealt with.